.github/CODEOWNERS

```
@@ -82,3 +82,4 @@
 /src/yarpviz/**                                  @Nicogene
 CMakeLists.txt                                   @drdanz
 *.cmake                                          @drdanz
+scripts/pip                                      @dferigo
```

I might be wrong here, but I think that you might need to do this:
```suggestion
scripts/pip/**                                   @dferigo
```

---

Right, done in https://github.com/robotology/yarp/compare/cb1f143b3deeabdd0c9cdafacc37f56222d8bfa8..27d14b3300ed3bee0824c5b0a8e0b4c6608fa7bd